realm/realm-library/src/main/cpp/objectserver_shared.hpp

```diff
@@ -59,6 +59,11 @@ class JniSession {
             g_vm->AttachCurrentThread(&local_env, nullptr);
             std::string log = num_to_string(error_code) + " " + message.c_str();
             log_message(local_env, log_debug, log.c_str());
```

Comment:
you are right, there is no need to log it here. It has been logged in the sync as well.